### PR TITLE
add bg choice to defaults json.

### PR DESCRIPTION
**Pull request by the-j0k3r**
Some background:

Following https://github.com/StylishThemes/StackOverflow-Dark/issues/188 added background choice to the usercss.

 I have been struggling with [whacking that in default.json.](https://github.com/StylishThemes/StackOverflow-Dark/commit/9f52c47dd0bc669c18a5813e49a87e0b56a668f7)

This PR adds such config that without running build script will remove that usercss entry. so we add those the defaults.json however there's one issue and one caveat.

1) issue is this adds quotes to before and after { } this breaks the resulting CSS.
2) the dropdown menu entries are all lowercase cant figure out how to add upper case to the entries.
    You can seee the code that was replaced how it looks OK in usercss.

@silverwind @mottie for help

### CODE REVIEW

File: stackoverflow-dark.user.css [hunk 1]
```diff
@@ -11,18 +11,18 @@
...
 @advanced text font-choice "Code Font" Menlo
-@advanced dropdown bg-choice "Background" {
-  crosshatch "Cross-Hatch" <<<EOT
-  url(https://camo.githubusercontent.com/5b8e4cd1a5ce69b6affa17ebd8e4aee00c147618/68747470733a2f2f7374796c6973687468656d65732e6769746875622e696f2f4769744875622d4461726b2f696d616765732f6261636b67726f756e64732f62672d63726f737368617463682e706e67) EOT;
-  uniform "Uniform Tile" <<<EOT
-  url(https://camo.githubusercontent.com/f83a9a60a2c64be0c88833ccb436e32a458cd2ce/68747470733a2f2f7374796c6973687468656d65732e6769746875622e696f2f4769744875622d4461726b2f696d616765732f6261636b67726f756e64732f62672d74696c652d756e69666f726d2e706e67) EOT;
-  random "Random Tile" <<<EOT
-  url(https://camo.githubusercontent.com/3beb850d8069e424480199a09899030c9e58e85f/68747470733a2f2f7374796c6973687468656d65732e6769746875622e696f2f4769744875622d4461726b2f696d616765732f6261636b67726f756e64732f62672d74696c65312e706e67) EOT;
-  custom "Custom" <<<EOT
+@advanced dropdown bg-choice "Background" "{
```
Comment (the-j0k3r) [inline stackoverflow-dark.user.css]: its adding the `"` to before the `{`

File: stackoverflow-dark.user.css [hunk 2]
```diff
@@ -11,18 +11,18 @@
...
+  url('https://camo.githubusercontent.com/f83a9a60a2c64be0c88833ccb436e32a458cd2ce/68747470733a2f2f7374796c6973687468656d65732e6769746875622e696f2f4769744875622d4461726b2f696d616765732f6261636b67726f756e64732f62672d74696c652d756e69666f726d2e706e67') EOT;
+  random "random" <<<EOT
+  url('https://camo.githubusercontent.com/3beb850d8069e424480199a09899030c9e58e85f/68747470733a2f2f7374796c6973687468656d65732e6769746875622e696f2f4769744875622d4461726b2f696d616765732f6261636b67726f756e64732f62672d74696c65312e706e67') EOT;
+  custom "custom" <<<EOT
   var(--bg-custom) EOT;
-  none "None" <<<EOT
+  none "none" <<<EOT
   none EOT;
-}
+}"
```
Comment (the-j0k3r) [inline stackoverflow-dark.user.css]: its adding the `"` to after the `}`

**Comment by the-j0k3r** [comment]
ping help please =)

**Comment by Mottie** [comment]
Ok, try that fix

**Comment by the-j0k3r** [comment]
Works cant believe I missed the quotes =(

Thanks.

**Pull request merged**